src/Release/CreateRelease.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Clearbooks\Labs\Release;
+use Clearbooks\Labs\Release\Gateway\ReleaseGateway;
+use Clearbooks\Labs\Release\UseCase\CreateRelease\Request;
+use Clearbooks\Labs\Release\UseCase\CreateRelease\Response;
+use Clearbooks\Labs\Release\CreateRelease\ResponseModel;
+
+/**
+ * @author: Ryan Wood <ryanw@clearbooks.co.uk>
+ * @created: 15/07/15
+ */
+class CreateRelease implements \Clearbooks\Labs\Release\UseCase\CreateRelease
+{
+    /**
+     * @var ReleaseGateway
+     */
+    private $releaseGateway;
+
+    /**
+     * @param ReleaseGateway $releaseGateway
+     */
+    public function __construct( ReleaseGateway $releaseGateway )
+    {
+        $this->releaseGateway = $releaseGateway;
+    }
+
+    /**
+     * @param Request $request
+     * @return Response
+     */
+    public function execute( Request $request )
+    {
+        $errors = $this->validateRequest( $request );
+
+        $response = $this->getResponse( $errors );
+
+        if( !$response->isSuccessful() ) {
+            return $response;
+        }
+
+        $response->setId( $this->releaseGateway->addRelease( $request->getReleaseName(), $request->getReleaseInfoUrl() ) );
```

Comment:
I figured as the DB stuff for labs is going to be in a different repo it would be classed as "3rd party" therefore we need a gateway class to access it, rather than a storage class that would interact with a storage server directly. Got this from the clearbooks wiki on OBI framework. 
![image](https://cloud.githubusercontent.com/assets/8090902/8904038/23b36716-3454-11e5-9f6d-24da953ff97a.png)